arduino/final_project/bluetooth.h

```diff
@@ -44,7 +51,6 @@ void gsend_ms(const char& msg) // send command to python
 {
```

Comment:
we don't need to send message to python. I will write a function to send rfid to python.